src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -935,25 +873,22 @@ private static LockedConfigurations computeConfigurationsToLock(Project project,
 
         ImmutableLockedConfigurations.Builder lockedConfigurations = ImmutableLockedConfigurations.builder();
 
-        lockedConfigurations.addAllProductionConfigurations(Collections2.transform(
-                ext.getProductionConfigurations(),
-                project.getConfigurations()::getByName));
-        lockedConfigurations.addAllTestConfigurations(Collections2.transform(
-                ext.getTestConfigurations(),
-                project.getConfigurations()::getByName));
+        lockedConfigurations.addAllProductionConfigurations(
+                Collections2.transform(ext.getProductionConfigurations(), project.getConfigurations()::getByName));
+        lockedConfigurations.addAllTestConfigurations(
+                Collections2.transform(ext.getTestConfigurations(), project.getConfigurations()::getByName));
 
         if (ext.isUseJavaPluginDefaults() && project.getPluginManager().hasPlugin("java")) {
             SourceSetContainer sourceSets =
                     project.getConvention().getPlugin(JavaPluginConvention.class).getSourceSets();
 
-            lockedConfigurations.addAllProductionConfigurations(getConfigurationsForSourceSet(
-                    project,
-                    sourceSets.getByName(SourceSet.MAIN_SOURCE_SET_NAME)));
+            lockedConfigurations.addAllProductionConfigurations(
+                    getConfigurationsForSourceSet(project, sourceSets.getByName(SourceSet.MAIN_SOURCE_SET_NAME)));
 
             // Use heuristic for test source sets.
-            sourceSets.matching(sourceSet -> sourceSet.getName().toLowerCase().endsWith("test"))
-                    .forEach(sourceSet -> lockedConfigurations
-                            .addAllTestConfigurations(getConfigurationsForSourceSet(project, sourceSet)));
+            sourceSets.matching(sourceSet -> sourceSet.getName().toLowerCase().endsWith("test")).forEach(
+                    sourceSet -> lockedConfigurations.addAllTestConfigurations(
+                            getConfigurationsForSourceSet(project, sourceSet)));
```

Comment:
This is not great but can be avoided if we don't allow breaking at method arguments unless the last argument is also breakable (e.g. a lambda / new anonymous class / binary operator)